File: Phyton/Sequencial/Terreno.py
largura = float(input("Digite a largura do terreno: "))
comprimento = float(input("Digite o comprimento do terreno: "))
metroQuadrado = float(input("Digite o valor do metro quadrado: "))

area = largura*comprimento
valor = area * metroQuadrado

print(f"Area do terreno = {area}")
print(f"Preço do terreno = {valor}")

File: C/Sequencial/Terreno.c
#include <stdio.h>

int main()
{
    double lag, comp, valor, area, preco;

    printf("Digite a largura do terreno: ");
    scanf("%lf", &lag);

    printf("Digite a comprimento do terreno: ");
    scanf("%lf", &comp);

    printf("Digite o valor do metro quadrado:");
    scanf("%lf", &valor);

    area = lag * comp;
    preco = area * valor;

    printf("Area do terreno = %.2lf\n", area);
    printf("Preco do terreno = %.2lf\n", preco);






return 0;
}
